platform-api-client-gwt/che-core-client-gwt-machine/src/main/java/org/eclipse/che/api/machine/gwt/client/MachineServiceClient.java

```diff
@@ -73,6 +74,14 @@
     Promise<List<MachineDescriptor>> getMachines(@Nullable String projectPath);
 
     /**
+     * Find processes related to the specified machine.
+     *
+     * @param machineId
+     *         machine id which allows defines needed machine
```

Comment:
strange description )
I guess something like: "id of machine to get processes information from" will be better